docs/release_notes/release-2.5.md

```diff
@@ -1,5 +1,30 @@
 # StarRocks version 2.5
 
+## 2.5.13
+
+Release date: September 28, 2023
+
+### Improvements
+
+- Window functions COVAR_SAMP, COVAR_POP, CORR, VARIANCE, VAR_SAMP, STD, and STDDEV_SAMP now support the ORDER BY clause and Window clause. [#31352](https://github.com/StarRocks/starrocks/pull/31352)
+- An error instead of NULL is returned if a decimal overflow occurs during queries on the DECIMAL type data. [#30419](https://github.com/StarRocks/starrocks/pull/30419)
+- Executing SQL commands with invalid comments now returns results consistent with MySQL. [#30210](https://github.com/StarRocks/starrocks/pull/30210)
+- Rowsets corresponding to tablets that have been deleted are cleaned up, reducing the memory usage during BE startup. [#30625](https://github.com/StarRocks/starrocks/pull/30625)
```

Comment:
shall we use the backport pr number in the release branch or the pr in main branch?

Reply:
By default, I would use the original PR number as it may have important details in the conversation, but if the backport is modified because of a conflict or different functionality in the lower version then using the default could be misleading. So, for 30625 the change is really "increase the `load_tablet_timeout_seconds` default to 60" I think and link goes to #30656 if I am reading the backport correctly.

Reply:
I am not questioning this specific #30656 (it was just a randomly chosen one), but would understand the generic principles on how we choose the PR number from main branch rather than the target release branch. It is more straight forward to use the target pr number from code perspective, because the PR link is provided in the release note, and if the user is interested, he or she can click the link and goes directly to the corresponding PR. In such case, if the PR number or link is provided on main branch, confusion are introduced, because the PR is target to main branch, how's that connected to a release fixes, unless the user fully understand how the backport is running in this StarRocks project.

Reply:
I get it, I was just looking for one with a merge conflict as that was the only reason I could think of for using the target branch PR instead of the original.  I default to using the original because it is one less click for the reader to get to the conversation around the PR; but it is misleading for the people who prefer to look at the code for the change.

@EsoragotoSpirit what is the standard way of doing this on the team?

Reply:
Just come up with a different viewpoint of this thing. One way or the other, you guys decide which is the better choice. :)